specificatie/kadaster-natuurlijk-personen.yaml

```diff
@@ -9,7 +9,7 @@ paths:
     get:
       operationId: GetKadasterPersoon
       description: |
-                    Het raadplegen van een bij het kadaster geregistreerde natuurlijke persoon die niet in de basisregistratie personen (BRP) is ingeschreven (of wel is ingeschreven maar niet is gekoppeld bij het inschrijven van de akte). Kadasternatuurlijkpersonen worden niet geactualiseerd.
+                    Raadpleeg een bij het kadaster geregistreerde natuurlijke persoon die niet in de basisregistratie personen (BRP) is ingeschreven, of wel is ingeschreven maar niet is gekoppeld bij het inschrijven van de akte. Kadasternatuurlijkpersonen worden niet geactualiseerd.
```

Comment:
Elke zin in een eigen regel? Dit gebeurt soms wel en soms niet. Is wel makkelijker bij het lezen. Dan hoeft je niet heen en weer te scrollen